consensus/bor/bor.go

```diff
@@ -816,14 +816,16 @@ func (c *Bor) Finalize(chain consensus.ChainHeaderReader, header *types.Header,
 
 	headerNumber := header.Number.Uint64()
 
-	if len(withdrawals) > 0 {
+	if withdrawals != nil {
+		//nolint
+		withdrawals = nil
 		log.Error("Bor does not support withdrawals", "number", headerNumber)
```

Comment:
Let's change all logs to `Warn` as we don't want to act on them.

Reply:
+1